create-lint-config/package.json

```diff
@@ -6,7 +6,6 @@
   "author": "Alex LaFroscia <alafroscia@movableink.com>",
   "repository": "https://github.com/movableink/lint-config",
   "license": "MIT",
-  "private": false,
```

Comment:
Yarn removed these when upgrading to version 3. Not sure why, but it probably doesn't hurt, either...